.github/workflows/ci_tests_legacy.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Install dependencies
         run: |
           mamba install gmt=${{ matrix.gmt_version }} numpy \
-                        pandas xarray netCDF4 packaging geopandas \
+                        pandas xarray netCDF4 packaging contextily geopandas \
```

Comment:
Note to self: fix merge conflict with https://github.com/GenericMappingTools/pygmt/pull/2283 later which will add in `ipython`.